deploy/k8s/eks/Readme.md

```diff
@@ -0,0 +1,66 @@
+# Setup Eks clusters with eksctl
+
+[eksctl](https://eksctl.io/) is a simple CLI tool for creating and managing clusters on EKS - Amazon's managed Kubernetes service for EC2. It is written in Go, uses CloudFormation, was created by Weaveworks and it welcomes contributions from the community. Create a basic cluster in minutes with just one command.
+
+
+## Installation
+
+### Prerequisites
+Follow guidelines below if not yet installed.See [references](#useful-references) section below for links to installation.
+
+1. **AWS CLI**
+
+   You will need to have AWS CLI credentials configured. You can use [`~/.aws/credentials` file][awsconfig]
+   or [environment variables][awsenv]. For more information read [AWS documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-environment.html).
+
+   [awsenv]: https://docs.aws.amazon.com/cli/latest/userguide/cli-environment.html
+   [awsconfig]: https://docs.aws.amazon.com/cli/latest/userguide/cli-config-files.html
+
+   You will also need [AWS IAM Authenticator for Kubernetes](https://github.com/kubernetes-sigs/aws-iam-authenticator) command (either `aws-iam-authenticator` or `aws eks get-token` (available in version 1.16.156 or greater of AWS CLI) in your `PATH`.
```

Comment:
Is installing the IAM authenticator a 'must'? 

I think that the `eksctl` can work without it, no?

Reply:
If installed via brew , it will include iam-Authenticator as a dependency.

Reply:
So just to verify that I understand - the user can just run `brew install weaveworks/tap/eksctl`, instead of doing the second step?

If that's the case, I think that for the user it would be better to have something like:

Skip this step if using homebrew 
2. Kubectl........

If you have another suggestion, I am also good with it. I just want it will be very clear for the user that he can skip this step if using homebrew without need to go into too much details.

Reply:
Added in bold to skip 1&2